tools/profiling/dbpreader.c

```diff
@@ -780,7 +780,8 @@ int dbp_reader_last_error(const dbp_multifile_reader_t *dbp)
 
 void dbp_reader_close_files(dbp_multifile_reader_t *dbp)
 {
-    for( int i = 0; i < dbp->nb_files; i++ ) {
```

Comment:
Indentation of `int i = 0;` should be aligned with the `for`.

Reply:
Do not rearrange other people's code. Remove this please.

Reply:
Sorry, that was accidental.